### ISSUE TITLE
TGFS is very slow 

### ISSUE THREAD

**Issue by petRUShka** [Firemoon777/tgfs#3]
I have quite a big chat in TG. I've mounted Telegram to folder with `tgfs`. And I've started `rsync --verbose -a --progress` and for whole night it was only 14 of ~1000 files that was copied!

So in that sense tgfs is unusable in that case...

**Comment by Firemoon777** [comment]
Yep, tgfs is slow, because it uses tg-cli, which can download only one file at one moment. So, it's kind of architecture bug in tgfs
